app/actions/preparations.py

```diff
@@ -0,0 +1,27 @@
+from app.people.knights import Knight
+
+
+def make_participants_list(participants: dict) -> dict:
+    participants_list = {}
+    for knight_name, knight_data in participants.items():
+        knight = Knight(
+            name=knight_data.get("name"),
+            power=knight_data.get("power"),
+            hp=knight_data.get("hp"),
+            armour=knight_data.get("armour"),
+            weapon=knight_data.get("weapon"),
+            potion=knight_data.get("potion"),
+        )
+        knight.preparation()
```

Comment:
u can call this method inside init method

Reply:
Moved